src/ui/public/styles/base.less

```diff
@@ -158,14 +158,16 @@ a {
         border-radius: 0;
       }
     }
+
     .kibana-nav-info {
       line-height: 30px;
       padding: 0 10px;
-      &-title {
+    }
+
+      .kibana-nav-info-title {
```

Comment:
I'm using the indentation to indicate a parent-child relationship with whitespace, without incurring the additional specificity of nesting. What do you think of this tactic? I've been on teams which have used it successfully in the past... it's a lot easier when the CSS has fewer nested selectors to begin with. :)